src/components/Header/Header.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
+import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
+import { ReactComponent as DopplerLogo } from '../../doppler-logo.svg';
+
+class Header extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      user: {},
+    };
+  }
+
+  componentWillMount() {
+    this.setState({ user: this.props.user })
+  }
+
+  render() {
+    return (
+      <header>
+        <DopplerLogo />
+        <HeaderNav />
+        <HeaderUserMenu user={this.state.user} />
+      </header>
+    );
+  }
 
-const Header = (props) => {
-  return (
-    <header>
-      <HeaderNav user={props.user} />
```

Comment:
I think that this line was ok.

Reply:
I have to research what user properties I need for the headernav yet

Reply:
👍